test/e2e/app-dir/navigation/navigation.test.ts

```diff
@@ -910,15 +912,6 @@ describe('app dir - navigation', () => {
         .click()
 
       if (!isNextDev) {
-        expect(
-          await browser
-            .waitForElementByCss(
-              '#loading',
-              // Give it some time to commit
-              100
-            )
-            .text()
-        ).toEqual('Loading')
```

Comment:
did we need to remove this? it seemed like it was verifying we rendered the loading state before the actual content resolved which seems useful to verify still

Reply:
there won't be loading anymore as it's not triggering `loading.js` since the metadata is suspended in body. we can assert there's no loading.js but probably not necessary?

Reply:
Oh right, I see! Ignore this then.